src/components/dialogs/payment.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-dialog max-width="45rem" :value="isVisible" persistent>
+  <v-dialog min-width="32rem" max-width="45rem" :value="isVisible" persistent>
```

Comment:
I was just playing around and found that setting the minimum width here really improves the UX on a small screen.